eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -185,10 +188,10 @@ function FindParentWorkItem($serviceName, $packageDisplayName, $outputCommand =
 $packageWorkItems = @{}
 $packageWorkItemWithoutKeyFields = @{}
 
-function FindLatestPackageWorkItem($lang, $packageName, $outputCommand = $true, $ignoreReleasePlannerTests = $true)
+function FindLatestPackageWorkItem($lang, $packageName, $outputCommand = $true, $ignoreReleasePlannerTests = $true, $tag = $null)
 {
   # Cache all the versions of this package and language work items
-  $null = FindPackageWorkItem $lang $packageName -includeClosed $true -outputCommand $outputCommand -ignoreReleasePlannerTests $ignoreReleasePlannerTests
+  $null = FindPackageWorkItem $lang $packageName -includeClosed $true -outputCommand $outputCommand -ignoreReleasePlannerTests $ignoreReleasePlannerTests -tag $tag
```

Comment:
Why are you passing ignoreReleasePlannerTests twice? I'm still not a big fan of this flag.

Reply:
I didn't understand the comment about duplicate flag. ignoreReleasePlannerTests is passed to either ignore or consider the work item created by release planner test instances along with work items created by production systems (work items without release planner app test). Tag param is passed only when we need to fetch a work item with a given tag, so it won't consider a work item created by production system. Otherwise, we might end up linking a package work item from test incorrectly under a product work item created from production.

Reply:
Nevermind, I missed read this I thought you passed `$ignoreReleasePlannerTests` twice but one is just the parameter name `-ignoreReleasePlannerTests`